### DISCUSSION TITLE
How does backup work?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#923] [App and O/S Config Questions]
After loosing an SD card and having to start over, I'm keen to make sure backup works.

When I try creating a backup, I get a message something like "insecure download blocked".  In my downloads folder, I see files named like a download in progress file.

I suspect I need to open up something in my firewall, but I'm not sure what needs to be done.

**Comment by tagyoureit** [comment]
The backup saves the files locally (on your SD card) and then uses the browser to download them.  You can try a different browser or SSH to the Pi and manually download the backup directory.
